### PR TITLE
[ETFE-2165] added the DestinationBusinessName controller/route/view

### PR DESCRIPTION
**Pull request by LadonJackson**

### CODE REVIEW

**Review by r-melvin — APPROVED** [review]

**Review by r-melvin — DISMISSED** [review]

File: app/views/sections/destination/DestinationBusinessNameView.scala.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    @formHelper(action = onSubmitCall) {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        @govukInput(
+            InputViewModel(
+                field = form("value"),
+                label = LabelViewModel(HtmlContent(heading))
```
Comment (james-work-account) [inline app/views/sections/destination/DestinationBusinessNameView.scala.html:43]:
```suggestion
                label = LabelViewModel(Text(messages("destinationBusinessName.heading"))).asPageHeading(LabelSize.Large)
```

File: app/views/sections/destination/DestinationBusinessNameView.scala.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+@(form: Form[_], onSubmitCall: Call)(implicit request: DataRequest[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("destinationBusinessName.title"))) {
+
+    @formHelper(action = onSubmitCall) {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
```
Comment (james-work-account) [inline app/views/sections/destination/DestinationBusinessNameView.scala.html:39]:
```suggestion

@h2(messages("section.destination"), "govuk-caption-xl")
```
(it'd be better if this is `destination.subHeading` to align with the others but that's just a nitpick, not a priority)

File: app/views/sections/destination/DestinationBusinessNameView.scala.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+@import models.requests.DataRequest
+@import viewmodels.InputWidth._
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukInput: GovukInput,
+    govukButton: GovukButton,
+    h1: components.h1,
```
Comment (james-work-account) [inline app/views/sections/destination/DestinationBusinessNameView.scala.html:26]:
```suggestion
    h1: components.h1,
    h2: components.h2,
```

File: app/views/sections/destination/DestinationBusinessNameView.scala.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.requests.DataRequest
+@import viewmodels.InputWidth._
+
```
Comment (james-work-account) [inline app/views/sections/destination/DestinationBusinessNameView.scala.html:19]:
```suggestion
@import viewmodels.LabelSize

```

File: app/views/sections/destination/DestinationBusinessNameView.scala.html [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+            .withWidth(Full)
+        )
+
+        @continueOrExit()
+    }
+}
+
+@heading = {
+    @h1("destinationBusinessName.heading", Some(messages("section.destination")), "govuk-heading-l")
+}
```
Comment (james-work-account) [inline app/views/sections/destination/DestinationBusinessNameView.scala.html:55]:
```suggestion
@{
    //$COVERAGE-OFF$
}
```

**Review by james-work-account — CHANGES_REQUESTED** [review]

**Review by james-work-account — APPROVED** [review]

**Review by r-melvin — APPROVED** [review]

**Pull request merged**